### ISSUE TITLE
Drilldown dropdowns are not totally keyboard navigable

### ISSUE THREAD

**Issue by marcoscv-work** [liferay/clay#4996]
Drilldown dropdowns are not totally keyboard navigable

https://user-images.githubusercontent.com/7963804/179739360-17a9a137-6a79-429e-b061-04eb2c283023.mov

## What are the steps to reproduce?

- Open https://clayui.com/docs/components/drop-down.html/#dropdownwithdrilldown
- Use keyboard to focus the dropdown and press enter
- User arrowkeys to move to the Subnav and press enter
- You can see now how you have lost the focus (we can press tab to recover it but users are not aware and it's not an standard the combination of arrows and tab at the same html widget)

## What is the expected result?

We should be able to use keys to move between navigation levels
